spec/datetimeformat.html

```diff
@@ -452,6 +556,26 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-Intl.DateTimeFormat.prototype.formatToParts">
+      <h1>get Intl.DateTimeFormat.prototype.formatToParts</h1>
+
+      <p>
+        Intl.DateTimeFormat.prototype.formatToParts is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:
```

Comment:
Similar to #79, should we make this a regular method and not an accessor?  We could then remove the binding mechanism from the `formatToParts` API here.